registry/storage/driver/gcs/gcs_test.go

```diff
@@ -70,14 +77,21 @@ func init() {
 		ts = jwtConfig.TokenSource(dcontext.Background())
 	}
 
+	gcs, err := storage.NewClient(dcontext.Background(), option.WithCredentialsJSON(jsonKey))
+	if err != nil {
+		panic(fmt.Sprintf("Error initializing gcs client : %v", err))
+	}
+
 	gcsDriverConstructor = func(rootDirectory string) (storagedriver.StorageDriver, error) {
 		parameters := driverParameters{
-			bucket:        bucket,
-			rootDirectory: root,
-			email:         email,
-			privateKey:    privateKey,
-			client:        oauth2.NewClient(dcontext.Background(), ts),
-			chunkSize:     defaultChunkSize,
+			bucket:         bucket,
+			rootDirectory:  root,
+			email:          email,
+			privateKey:     privateKey,
+			client:         oauth2.NewClient(dcontext.Background(), ts),
+			chunkSize:      defaultChunkSize,
+			GCS:            gcs,
+			maxConcurrency: uint64(8),
```

Comment:
FYI: [literal numbers in Go don't have a type.](https://go.dev/blog/constants#numbers)
```suggestion
			maxConcurrency: 8,
```